recipes-bsp/firmware-imx/firmware-imx-9_8.18.bb

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 NXP
+SUMMARY = "Freescale i.MX firmware for i.MX 9 family"
+DESCRIPTION = "Freescale i.MX firmware for i.MX 9 family"
+
+require recipes-bsp/firmware-imx/firmware-imx-${PV}.inc
+
+inherit deploy
+
+do_install[noexec] = "1"
+
+do_deploy() {
+    # Synopsys DDR
+    for ddr_firmware in ${DDR_FIRMWARE_NAME}; do
+        install -m 0644 ${S}/firmware/ddr/synopsys/${ddr_firmware} ${DEPLOYDIR}
+    done
+}
+addtask deploy after do_install before do_build
+
+PACKAGE_ARCH = "${MACHINE_SOCARCH}"
+
+COMPATIBLE_MACHINE = "(mx9-nxp-bsp)"
```

Comment:
Why is this file required? It is a copy of `firmware-imx-8m`, which can be used in this context as well.

Reply:
This file could be removed, right?

Reply:
I'm currently testing this

Reply:
I got it working somehow, I will update the code so you can review. After review I can test other SoCs

Reply:
@otavio, wouldn't it be better to combine all these different `firmware-imx` recipes since they are all using the same `SRC_URI`?

Reply:
> @zandrey it is no the copy of firmware-imx-8m. recently, firmware-imx-9 is for i.MX 9 series firmware only.


From the recipe content perspective, it is a copy.

I tend to agree with @thochstein comment above to combine `firmware-imx` into one recipe, since `8m` and `9` in this context point to the same `SRC_URI`.